packages/client/src/store/userStore.ts

```diff
@@ -118,8 +119,10 @@ export const useUserStore = create<IUserStore>()(
               isAuth: true,
             },
           }));
-        } catch (error: unknown) {
-          addToast((error as IEError).response.data.reason);
+        } catch {
+          set(() => ({
+            user: initialUser,
+          }));
```

Comment:
Эти действия очень похожи на logout. Лучше не дублировать логику (в случае изменения метода, придется искать все места его применения и опять же дублировать дополнительную логику). Поэтому предлагаю следующий вариант:
```suggestion
          get().logout() // функция get возвращает стор внутри стора
```

Reply:
Outdated